var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage, ROCmPackage):
+    """ReSolve is a library of GPU-resident sparse linear solvers. It contains iterative and direct
+    solvers designed to run on NVIDIA and AMD GPUs, as well as CPU devices."""
+
+    homepage = "https://github.com/ORNL/ReSolve"
+    url = "https://github.com/ORNL/ReSolve.git"
+    maintainers("cameronrutherford", "pelesh", "ryandanehy", "kswirydo")
+
+    # version("1.0.0", submodules=False, branch=develop)
+    # version("develop", submodules=False, branch=develop)
```

Comment:
```suggestion
    version("develop", submodules=False, branch="develop")
```
@balay is this the change you are suggesting?

Reply:
yes - and also `git = ` as listed in that diff...